tasks/kernel_matrix_testing/vmconfig.py

```diff
@@ -77,6 +78,9 @@
     "amzn_4.14": "amzn_4.14",
     "amzn_5.4": "amzn_5.4",
     "amzn_5.10": "amzn_5.10",
+    "amzn_3": "amzn_3",
+    "amazon_2023": "amzn_3",
+    "al3": "amzn_3",
```

Comment:
`amzn_3`? It is called Amazon Linux 2023 and not Amazon Linux 3

Reply:
changed to `amzn_2023`